docs/about/introduction.adoc

```diff
@@ -16,102 +16,120 @@
-xref:../se/webclient.adoc[Learn more about the Helidon Web Client].
+- You want to base your application on modern enterprise Java standards such as Jakarta EE and MicroProfile
+- You are familiar with Java EE, Jakarta EE or Spring Boot and would like to use similar development experience
+- You are migrating existing Java EE/Jakarta EE application to microservices
+- You are planning to use CDI components or extensions
+- You are planning to use JPA for data access and Jersey (JAX-RS) for RESTful services
 
-* *Additional Websocket Support* +
-Based upon the Tyrus implementation, Helidon receives WebSocket API support.
-xref:../se/websocket.adoc[Learn more about Websocket Support].
 
-* *Support for Java 17 APIs* +
-Helidon will require Java 17 or newer.
-xref:prerequisites.adoc[Learn more about the prerequisites for Helidon 2.0].
+[NOTE]
+====
+If you don't know which Helidon flavor to use – use *Helidon MP*
+====
 
-* *CORS support for MP and SE* +
-Although it is possible for any Helidon application to implement its own support for CORS, there are common tasks
- (such as processing preflight requests) that can be provided in a Helidon module.
- xref:../se/cors.adoc[Learn more about CORS support].
+== Prerequisites
 
-* *Backward Incompatible Changes* +
-View the following links for information about potential breaking changes, including package name changes:
+Helidon 3 requires Java 17 (or newer) and Maven. You need Docker if you want to build and deploy Docker containers. If you want to deploy to Kubernetes, you need `kubectl` and a Kubernetes cluster (you can xref:kubernetes.adoc[install one on your desktop]).
 
-** https://github.com/oracle/helidon/blob/2.0.0/CHANGELOG.md#backward-incompatible-changes-1[Backward incompatible changes in 2.0.0-RC2]
-** https://github.com/oracle/helidon/blob/2.0.0/CHANGELOG.md#backward-incompatible-changes-2[Backward incompatible changes in 2.0.0-RC1]
-** https://github.com/oracle/helidon/blob/2.0.0/CHANGELOG.md#backward-incompatible-changes-3[Backward incompatible changes in 2.0.0-M3]
-** https://github.com/oracle/helidon/blob/2.0.0/CHANGELOG.md#backward-incompatible-changes-4[Backward incompatible changes in 2.0.0-M2]
-** https://github.com/oracle/helidon/blob/2.0.0/CHANGELOG.md#backward-incompatible-changes-5[Backward incompatible changes in 2.0.0-M1]
+- https://www.oracle.com/technetwork/java/javase/downloads[Java{nbsp}SE{nbsp}17] (http://jdk.java.net[Open{nbsp}JDK{nbsp}17]) or newer
+- https://maven.apache.org/download.cgi[Maven 3.8.6+]
+- https://docs.docker.com/install/[Docker 18.09+]
+- https://kubernetes.io/docs/tasks/tools/install-kubectl/[Kubectl 1.16.5+]
+
+We are also strongly suggesting installing xref:cli.adoc[Helidon CLI] (Command Line Interface) which helps generating and building Helidon projects.
 
 == Next Steps
 
-Try the xref:prerequisites.adoc[Helidon Quickstart Examples].
+Next would be to choose a Helidon flavor and start using it. Check the following 
```

Comment:
This should point to a top level "Get Started" page like we do in 2.x (. See my comment on `sitegen.yaml`). For example, replace this section with just:

Try the xref:{rootdir}/about/getstarted.adoc[Helidon Quickstart Examples]

Reply:
We are going in direction of separating Helidon documentation into self-contained documentation for each flavor. My idea is that all QuickStart pages must be flavor specific. Having one page which explains how to add JAVA_HOME and contains links to SE and MP quickstart guides doesn't make sense to me. Users will go to SE or MP introductio and find proper QuickStart examples there.

Reply:
The point is: we should have *one* page we can link to for getting started.  We need the "Get Started" button on the web site to go to one page. That one page might simply link to the other getting started pages -- but we need  one page. And if it is a top level, highly visible page all the better. This was a lesson we learned in 2.x. I would rather not repeat the same mistake again.

Reply:
Get Started page is back!